frontend/src/modules/resourceprogress/components/ResourceProgress.css

```diff
@@ -135,4 +135,5 @@
     font-size: 22px;
     font-weight: 300;
     padding: 5px 0 10px;
+    cursor: pointer;
```

Comment:
This is not needed now (because the elements are links, so the pointer automatically gets turned into a pointer).